src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi

```diff
@@ -125,7 +125,7 @@ cdef class _AioCall(GrpcCallWrapper):
         if credentials is not None:
             set_credentials_error = grpc_call_set_credentials(self.call, credentials.c())
             if set_credentials_error != GRPC_CALL_OK:
-                raise Exception("Credentials couldn't have been set")
+                raise RuntimeError(f"Credentials couldn't have been set: {set_credentials_error}")
```

Comment:
Retain on using F strings since we have to give support for Python 3.5.X users

Reply:
No worries, I can change it back, but
1. We are targetting ``Python3.6+``
2. Python 2 is no longer supported, I don't think that should be a constraint